pkg/features/kube_features.go

```diff
@@ -682,7 +682,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	TokenRequest:                   {Default: true, PreRelease: featuregate.Beta},
 	TokenRequestProjection:         {Default: true, PreRelease: featuregate.Beta},
 	BoundServiceAccountTokenVolume: {Default: false, PreRelease: featuregate.Alpha},
-	ServiceAccountIssuerDiscovery:  {Default: false, PreRelease: featuregate.Alpha},
+	ServiceAccountIssuerDiscovery:  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
add `// beta: v1.20` to the `ServiceAccountIssuerDiscovery` definition above

Reply:
Done

Reply:
nit: keep the alpha line intact and add the beta line so we can see progression at a glance

Reply:
Done